repose-aggregator/components/filters/valkyrie-authorization/src/test/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilterTest.scala

```diff
@@ -829,6 +829,162 @@ class ValkyrieAuthorizationFilterTest extends FunSpec with BeforeAndAfter with M
       assert((json \ "metadata" \ "count").asInstanceOf[JsNumber].as[Int] == 2)
     }
 
+    it("should remove null values") {
```

Comment:
800 lines? Probably should separate some of this stuff into multiple files with related content, and refactor out reused bits into a trait?